client/verta/verta/registry/entities/_model.py

```diff
@@ -13,6 +13,8 @@
 from .. import _constants, VertaModelBase
 from ._modelversion import RegisteredModelVersion
 from ._modelversions import RegisteredModelVersions
+from .. import action_type as action
+from .. import data_type as data
```

Comment:
`action` and `data` are common variable names, so we should make these more unique to avoid accidentally reusing them

```suggestion
from .. import action_type as action_type_module
from .. import data_type as data_type_module
```